convene-web/Gemfile

```diff
@@ -34,42 +34,42 @@ gem 'jbuilder', '~> 2.7'
 # View Layer
 #
 # Allows us to render .markdown.erb files
-gem 'redcarpet'
+gem 'redcarpet', '~> 3.5'
 # Breadcrumbs!
-gem 'gretel'
+gem 'gretel',  '~> 4.2'
 
 # Database Layer
 #
 # Postgres extensions for ActiveRecord
 # @see https://github.com/GeorgeKaraszi/ActiveRecordExtended
-gem 'active_record_extended'
+gem 'active_record_extended', '~> 2.0'
 # Slug-based model lookup
-gem 'friendly_id', '~> 5.2.4'
+gem 'friendly_id', '~> 5.2'
 # Support Active Model has_secure_password
-gem 'bcrypt', '~> 3.1.7'
+gem 'bcrypt', '~> 3.1'
 # Use postgresql for data persistence
-gem 'pg', '>= 0.18', '< 2.0'
+gem 'pg', '~> 1.2'
 
 # User-facing features
 #
 # Allows logging in via magic links
-gem 'passwordless'
+gem 'passwordless', '0.10.0'
 # Provides powerful administration features
-gem 'administrate'
+gem 'administrate', '0.16.0'
```

Comment:
Why lock this one to 0.16.0?

Reply:
Because the version number is less than 0, semver does not have guarantee that patch or minor version bumps will not break the APIs it exposes.